plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -186,19 +186,18 @@ def _handle_erratum(repo, type_id, unit_key, metadata, file_path, conduit, confi
     unit_data.update(metadata or {})
     unit_data.update(unit_key or {})
 
-    existing_unit = model_class.objects.filter(**unit_key).first()
-    new_unit = model_class(**unit_data)
+    unit = model_class(**unit_data)
+    unit._repo_id = repo.repo_id
```

Comment:
Can you elaborate on why this is being saved on the errata model? I know it used to be stored on each collection within it, but I'm not following why we would store it on an Errata.